src/types/index.d.ts

```diff
@@ -3,6 +3,14 @@ import { Wallet } from "ethers";
 type Network = "homestead" | "ropsten" | "rinkeby" | "local";
 type FormType = "TRANSFERABLE_RECORD" | "VERIFIABLE_DOCUMENT";
 
+export interface PublishState {
```

Comment:
Use enum, not interface, there isn't anything that implements this interface.

Reply:
ok removed